roles/edpm_ovn/defaults/main.yml

```diff
@@ -1,5 +1,6 @@
 ---
 # defaults file for edpm_ovn
+edpm_ovn_config_src: /var/lib/openstack/configs
```

Comment:
the actual file path includes `ovn`, e.g. `/var/lib/openstack/configs/ovn/ovsdb-config`

Reply:
correct, the "ovn" comes from the service name on the dataplane-operator side